modules/ROOT/nav.adoc

```diff
@@ -14,6 +14,9 @@
 **** xref:laravel-tiny-cloud.adoc[Laravel]
 **** xref:rails-cloud.adoc[Ruby on Rails]
 *** Self-hosted
+**** JavaScript
```

Comment:
This should not be here... these pages already exist as `zip-install.adoc` and `npm-projects.adoc`

Reply:
Right, okay we can leave this here as a placeholder to fix

Reply:
There's nothing to fix here... those are already there you just need to delete/revert these changes. Also remember that most of these are JavaScript, so having a heading like this would only be confusing.

Reply:
This will also need to be reverted in the tinymce source code aswell, will follow that up later, leaving as is for now